samples/bluetooth/central_otc/src/main.c

```diff
@@ -592,12 +616,19 @@ static void on_obj_data_written(struct bt_ots_client *ots_inst, struct bt_conn *
 	printk("Object been written %d\n", len);
 }
 
+void on_obj_checksum_got(struct bt_ots_client *ots_inst,
+			 struct bt_conn *conn, int err, uint32_t crc)
+{
+	printk("Object checksum OACP result (%d) crc 0x%08x\n", err, crc);
```

Comment:
Shouldn't we do our own local checksum to compare here? A checksum without a comparison is rather useless :)

Reply:
Yes, I will add comparison local and remote checksum here.

Reply:
Done.